src/test/java/steps/WebSteps.java

```diff
@@ -0,0 +1,41 @@
+package steps;
+
+import factoryPattern.PageFactory;
+import factoryPattern.PageTypeEnums;
+import factoryPattern.TestedPage;
+import net.serenitybdd.core.pages.WebElementFacade;
+import net.thucydides.core.annotations.Step;
+import org.openqa.selenium.Keys;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class WebSteps extends AbstractSteps {
+
+    @Step
+    public void openPageByParameter(final String nameOfPage) {
+        PageFactory.createPage(PageTypeEnums.PageType.valueOf(nameOfPage)).openTestedPage();
+    }
+
+    @Step
+    public void typeTextToInputTextBox(final String searchedText, final String elementName, final String page) {
+        WebElementFacade elementUnderTest = getElementUnderTest(elementName, page);
+        elementUnderTest.type(searchedText);
+        elementUnderTest.sendKeys(Keys.ENTER);
+    }
+
+    @Step
+    public void userCheckElementContainsText(final String elementName, final String searchedText, final String page) {
```

Comment:
use PageTypeEnums instead of String type

Reply:
fixed